src/web/app/pages-instructor/view-photo-popover/view-photo-popover.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+
+/**
+ * Component to display a popover for photos
+ */
+@Component({
+  selector: 'tm-view-photo-popover',
+  templateUrl: './view-photo-popover.component.html',
+  styleUrls: ['./view-photo-popover.component.scss'],
+})
+export class ViewPhotoPopoverComponent implements OnInit {
+
+  @Input()
+  photoUrl: string = '';
+
+  @Input()
+  isContentAvailable: boolean = true;
+
+  @Output()
+  loadPhotoEvent: EventEmitter<any> = new EventEmitter();
```

Comment:
I am not sure why do we need this event for parent to handle.

Parent can just pass the full `photoUrl`. There is a new attribute in the component called `isPhotoShown`. Once the `View photo` is clicked, the attribute change and the photo is shown.

Reply:
My initial thought was to have the event so we can sync up all of the popovers. 

For example, in the `gqr-rqg` component, it has a `per-question-view-response` component which also has the popover.  I tried this feature in V6 and if you click on the view photo of a user in the `per-question` view, the header with the user name will also show the photo instead of the view photo link like shown below:

![gqr](https://user-images.githubusercontent.com/28994607/83216086-0559de80-a19b-11ea-974b-99220deef34c.gif)


So i thought we needed some way for the popover in `per-question` view to communicate with the popover in the `gqr-rqg` parent component. Thus, i used the event. 

Would there perhaps be a simpler way to do this? If we just change the attribute to `isPhotoShown` we still need to sync up the different popovers in the page right?

Reply:
I think the simplest way is to remove the sync-up LOL. 

Justification: `photoUrl` is not some resources requires explicit REST loading or form sync up. Therefore, it is a bit strange to pop up the event to the root page for loading.

A simpler implementation: keep a `Set` at root page to indicates which email get loaded. Pass the set to the child components. Child components fire events when the photo get loaded. There is no need to generate the URL at root page.

What do you think?

Reply:
Hmm alright, i think we can just remove the sync-up. Can always add it back in the future if we need the syncing 👍